packages/features/bookings/lib/handleCancelBooking.ts

```diff
@@ -89,6 +89,13 @@ async function handler(input: CancelBookingInput) {
     arePlatformEmailsEnabled,
   } = input;
 
+  if (bookingToDelete.status === BookingStatus.CANCELLED) {
```

Comment:
I think we should document here in code, the important reason to not allow Cancelling an already cancelled booking.

Because a booking could have been CANCELLED due to reschedule in which case we just update the existing calendar event and meeting, we want to prevent them being deleted